spec/peep_spec.rb

```diff
@@ -0,0 +1,29 @@
+require 'peep'
+require_relative 'database_helper'
+
+describe Peep do
+  describe '.create' do
+    it 'creates a new peep' do
+      msg = 'Coding is fun!'
+      peep = Peep.create(peep: msg)
+      data = data(table: 'peeps', id: peep.id)
+      expect(peep.id).to eq(data['id'])
+      expect(peep.peep).to eq(msg)
+      expect(peep.time).to match(/(\d{4})-(\d{2})-(\d{2}) (\d{2}):(\d{2}):(\d{2})/)
```

Comment:
You could make use of a mock here for the time so that you can assert that `peep.time` gives a timestamp.